libraries/default.rb

```diff
@@ -678,6 +678,29 @@ def install_multitenancy_config(current_deploy_root, admin_hostname, engage_host
       end
     end
 
+    def install_default_tentant_config(current_deploy_root, public_dns_name, private_dns_name)
+      lti_oauth_info = get_lti_auth_info
+      template %Q|#{current_deploy_root}/etc/security/mh_default_org.xml| do
+        source 'mh_default_org.xml.erb'
+        owner 'matterhorn'
+        group 'matterhorn'
+        variables({
+          lti_oauth: lti_oauth_info,
+          unproxied_name: public_dns_name,
```

Comment:
I initially thought I saw nginx change the DNS/IP in addition to the protocol. However, the host name was not changed when I tested on the latest deploys in my dev cluster. I left the split in here as a reminder that unproxied vs proxy host is significant, in case a forwarder changes this in the future (such as a load balancer). FYI - I hand jammed the "https" to "http" protocol change directly into the template, since I didn't know how to grab it from here.